src/test/java/uk/gov/companieshouse/certificates/orders/api/controller/CertificateItemsControllerIntegrationTest.java

```diff
@@ -273,7 +304,6 @@ void createCertificateItemSuccessfullyCreatesCertificateItem() throws Exception
         newItem.setCustomerReference(CUSTOMER_REFERENCE);
 
         final CertificateItemDTO expectedItem = new CertificateItemDTO();
-        expectedItem.setId(EXPECTED_ITEM_ID);
```

Comment:
We should probably still assert that the value of the id generated from the service, is in fact the id in the response for this test.

Reply:
Thanks, added back in